<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configure_Level_Form</class>
 <widget class="QDialog" name="Configure_Level_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Level Generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="layoutLevelScripts">
     <item>
      <widget class="QLabel" name="lblLevelScripts">
       <property name="text">
        <string>Level Scripts:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboLevelScripts"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearAllRandomLevelScripts">
       <property name="text">
        <string>Clear All Random Level Scripts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbGenerateNewLevels">
       <property name="text">
        <string>Generate New Levels</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutStandardOverworldLevels">
     <item>
      <widget class="QLabel" name="lblStandardOverworld">
       <property name="text">
        <string>Standard Overworld Levels:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboStandardOverworld"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutUndergroundLevels">
     <item>
      <widget class="QLabel" name="lblUnderground">
       <property name="text">
        <string>Underground Levels:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboUnderground"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutUnderwaterLevels">
     <item>
      <widget class="QLabel" name="lblUnderwater">
       <property name="text">
        <string>Underwater Levels:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboUnderwater"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutBridgeLevels">
     <item>
      <widget class="QLabel" name="lblBridge">
       <property name="text">
        <string>Bridge Levels:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBridge"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutIslandLevels">
     <item>
      <widget class="QLabel" name="lblIsland">
       <property name="text">
        <string>Island Levels:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboIsland"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
